### ISSUE TITLE
ttf-caladea-kagayaki: "cd caladea: No such file or directory"

### ISSUE THREAD

**Issue by simonbcn** [julroy67/kagayaki#3]
```
....
line 22: cd: caladea: No such file or directory
==> ERROR: A failure occurred in package().
    Aborting...
```

The right folder name is: `crosextrafonts-20130214`

**Comment by simonbcn** [comment]
And another error:
```
....
install: cannot stat 'truetype/*.ttf': No such file or directory
```

There isn't `truetype` folder.

**Comment by simonbcn** [comment]
This is the correct PKGBUILD:
```
# Maintainer: julroy67 <julroy67@gmail.com>
# Contributor: bohoomil <@zoho.com>

pkgname=ttf-caladea-kagayaki
_tarname=crosextrafonts
pkgver=20130214
pkgrel=1
pkgdesc="Caladeo is a serif font metrically compatible with Cambria"
arch=('any')
url="https://code.google.com/p/chromium/issues/detail?id=168879"
license=('Apache')
depends=('fontconfig')
conflicts=('ttf-caladea-ib')
replaces=('ttf-caladea-ib')
provides=('ttf-caladea-ib')
groups=('kagayaki-bundle-fonts')
source=(http://commondatastorage.googleapis.com/chromeos-localmirror/distfiles/crosextrafonts-20130214.tar.gz
        45-caladea.conf
        90-tt-caladea.conf)

package(){
	cd crosextrafonts-20130214

	install -m755 -d "${pkgdir}"/usr/share/fonts/"${pkgname}"
	install -m644 *.ttf "${pkgdir}"/usr/share/fonts/"${pkgname}"
	
	cd "${srcdir}"
	install -D -m644 45-caladea.conf \
		"${pkgdir}"/etc/fonts/conf.avail/45-caladea.conf
	install -D -m644 90-tt-caladea.conf \
		"${pkgdir}"/etc/fonts/conf.avail/90-tt-caladea.conf
	
	install -m755 -d "${pkgdir}"/etc/fonts/conf.d
	cd "${pkgdir}"/etc/fonts/conf.d
	ln -s ../conf.avail/45-caladea.conf .
	ln -s ../conf.avail/90-tt-caladea.conf .
}
sha1sums=('6f8f87cd21c4cd7e72031852eb90453dba3e2473'
          '90d29666e807c1f7456f4ffa86f4b6ed5b92639b'
          'd4747515ff61405209314f9d909975eec01f56bf')
```

**Comment by julroy67** [comment]
Fixed in 6b261ce58d66a225f5a27367f5bfc5825e793880
